rfdiffusion/diffusion.py

```diff
@@ -569,6 +571,7 @@ def __init__(
             b_T (float, required): Ending variance for Euclidean schedule
 
         """
+        self._log = logging.getLogger(__name__)
```

Comment:
Is there a reason to mix `logger` and `self._log`?

Reply:
Yeah the problem is that the place I'm using a module-level logger is a free function, so there's nowhere to attach the logger. RFdiffusion has this pattern of class-level loggers saved as `self._log`, so I've stuck to that where it works, but otherwise I mostly used the [recommended idiomatic python](https://docs.python.org/3/library/logging.html) pattern of module-level (I'm just realizing now that in a few functions I have function-level loggers).